src/hb-ot-os2-table.hh

```diff
@@ -125,6 +125,24 @@ struct os2
     *max_cp = max;
   }
 
+  enum font_page_t {
+    HEBREW_FONT_PAGE		= 0xB100, // Hebrew Windows 3.1 font page
+    SIMP_ARABIC_FONT_PAGE	= 0xB200, // Simplified Arabic Windows 3.1 font page
+    TRAD_ARABIC_FONT_PAGE	= 0xB300, // Traditional Arabic Windows 3.1 font page
+    OEM_ARABIC_FONT_PAGE	= 0xB400, // OEM Arabic Windows 3.1 font page
+    SIMP_FARSI_FONT_PAGE	= 0xBA00, // Simplified Farsi Windows 3.1 font page
+    TRAD_FARSI_FONT_PAGE	= 0xBB00, // Traditional Farsi Windows 3.1 font page
+    THAI_FONT_PAGE		= 0xDE00  // Thai Windows 3.1 font page
```

Comment:
Not that matters but the names of these should have been eg `FONT_PAGE_FARSI_SIMP` not other way around.

Reply:
Correct, makes sense

Reply:
https://github.com/harfbuzz/harfbuzz/commit/8f6d0f8401ffd011332981fd9ba6d38d18b6cf3b